mkt/latecustomization/views.py

```diff
@@ -41,24 +46,42 @@ class LateCustomizationViewSet(FeedShelfPermissionMixin, CORSMixin,
     filter_backends = [RegionCarrierFilter]
     cors_allowed_methods = ('get', 'post', 'delete')
 
-    def list(self, request, *args, **kwargs):
+    def get_carrier_region(self, request):
         data = self.req_data() or self.request.GET
-        carrier, region = data['carrier'], data['region']
+        if (('region' in data or 'carrier' in data) and
```

Comment:
Yeah it's redundant now, I originally had different error messages for the region/carrier and mnc/mcc cases but that all got smushed together here.